websites/G/Goodreads/presence.ts

```diff
@@ -1,7 +1,6 @@
 const presence = new Presence({ clientId: "867525909204566056" }),
-	timeElapsed: number = Math.floor(Date.now() / 1000);
-
-let book: string, author: string;
+	timeElapsed: number = Math.floor(Date.now() / 1000),
+	{ pathname, href } = document.location;
```

Comment:
Move inside of the `UpdateData` loop.

Reply:
I'll do this.

But, could you, please, explain why?

Reply:
It could keep the variables up to date if the page changes the path without triggering a page load. (such as using the history api, etc)

Reply:
I see